etc/syncd.d/msn2700_port_map.ini

```diff
@@ -0,0 +1,33 @@
+# alias          lanes
```

Comment:
I'd say it's better to have everything in the package. Otherwise why do we need this package? We can deploy our application using ansible only without any packages.